src/player_activity.cpp

```diff
@@ -113,9 +113,9 @@ void player_activity::do_turn( player &p )
     type->call_do_turn( this, &p );
 
     if( *this && type->rooted() ) {
-        p.rooted();
+        //p.rooted();
```

Comment:
What this check was doing was only giving the player their nutrition for being rooted while preforming activities with `"rooted": true` set in the json.

You can definitely remove this call to `rooted()`, as you now apply a rooted effect and don't require the player to be explicitly preforming an activity that allows rooting.  I *think* the `pause()` call should be kept though.

You do however now need to restrict the player to only being able to start activities with `"rooted": true`.